Documentation/adding-new-repository.md

```diff
@@ -41,6 +41,30 @@ When applicable, prepare Docker image repositories for the new component.
   * robots: Read & Write
 * For closed source repository, create a new private repository in registry.mender.io
 
+## Makefile
+
+Consider adding a Makefile to the repository. This will serve both developers on their
+local environment and automated templated jobs on CI.
+
+A Makefile in a Mender repository is expected to contain at least the following targets:
+
+* `fmt`: Format the code. Typically using `gofmt` for golang repositories
```

Comment:
"..or Black for Python files."

Reply:
I wrote "Format the code. Typically with `gofmt` for golang, `Black` for Python, and `clang-format` for C.".

For `mender-binary-delta` we don't check it via `Makefile` but directly on the pipeline script. But that can be a future improvement.